docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,244 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: knative-serving
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+EOF
+```
+
+Wait some time for Knative Serving is to become ready. Check the Knative Serving deployment with the following command:
+
+```
+kubectl get deployment -n knative-serving
+```
+
+You will receive the status of Knative serving deployments. If Knative Serving has been successfully deployed, you will
+see all the deployments of the Knative Serving have reached READY status.
+
+## Install Knative Eventing with Operator:
+
+All the releases for Knative Eventing Operator can be found on [this page](https://github.com/knative/eventing-operator/releases).
+
+### From releases:
+
+Replace \<version\> with the latest version or the version you would like to install, and run the following command to
+install Knative Eventing Operator:
+
+```
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/<version>/eventing-operator.yaml
+```
+
+### From source code:
+
+The source code is available on this page. You need to install the building tool [ko](https://github.com/google/ko) first. After downloading the source
+code, you can install the operator in the root directory of the source code with the following command:
+
+```
+ko apply -f config/
+```
+
+Verify the installation of Knative Eventing Operator with the command:
+
+```
+kubectl get deployment knative-eventing-operator
+```
+
+You should see the deployment is ready, if the operator is installed. Use the following command to track the log of
+the operator:
+
+```
+kubectl logs -f $(kubectl get pods -l name=knative-eventing-operator -o name)
+```
+
+If deployment of Knative Eventing operator is ready, it is time to install Knative Evenintg by installing the custom
+resource of Knative Eventing operator. The same to Knative Serving operator, you can use any namespace, but we recommend
+you to use `knative-eventing` for Knative eventing. In the rest of this tutorial, we keep on using `knative-eventing` as the
+namespace to create the Eventing operator CR and all the other namespaced Eventing resources. Run the following command:
+
+```
+cat <<-EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: knative-eventing
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+EOF
+```
+
+Wait some time for Knative Eventing to become ready. Check the Knative Eventing deployments with the following command:
+
+```
+kubectl get deployment -n knative-eventing
+```
+
+We should see a list of Knative Eventing deployments up and running. If Knative Eventing has been successfully deployed,
+you should see all the deployments of Knative Eventing are up and running.
+
+## Uninstall Knative Serving with Operator:
+
+### To remove Knative Serving component:
```

Comment:
Don't use headings this way please. Instead, something like:

### Removing the Knative Serving component
1. Remove the Knative Serving CR:
```
 kubectl delete KnativeServing knative-serving -n knative-serving
```